src/Lessons-api.Domain/TeacherModel/UpdateTeacherDTO.cs

```diff
@@ -1,6 +1,8 @@
-﻿namespace Lessons_api.Domain.TeacherModel
+﻿using Lessons_api.Domain.UserModel;
+
+namespace Lessons_api.Domain.TeacherModel
 {
-    public class UpdateTeacherDTO : BaseTeacherDTO
+    public class UpdateTeacherDTO : BaseUserDTO
```

Comment:
There are 3 models `TeacherDTO` `CreateTeacherDTO` and `UpdateTeacherDTO` 
We can replace all by one